### DISCUSSION TITLE
dock opacity/blur

### DISCUSSION THREAD

**Discussion by decide-you** [prasanthrangan/hyprdots#1871] [Q&A] [answered]
how can i change the opacity or disable blur for the dock? i know there are options 'opacity' and 'noblur' for windowrulev2, but what "class" do i need to change actually?

**Comment by kRHYME7** [comment, marked as answer]
@decide-you  the bar is a lay and it's namespace is "waybar" 

Layers behave differently.  
https://wiki.hyprland.org/Configuring/Window-Rules/#rules-1

**Reply by decide-you** [reply]
thanks! what i need - `layerrule = ignorezero, waybar`
